sqs_lambda/index.ts

```diff
@@ -53,6 +57,33 @@ mutation deleteUser($id: ID!) {
   }).then((response) => response.json());
 }
 
+/**
+ * Submit migrateAppleUser mutation POST request to client-api
+ * @param id FxA account ID to delete from Pocket's database
+ * @param email User email in the Fx event payload
+ * @param transferSub primary ID connecting fxa account and pocket account
+ */
+async function migrateAppleUserMutation(
+  id: string,
+  email: string,
+  transferSub: string
+): Promise<any> {
+  const privateKey = await getFxaPrivateKey();
+  const deleteMutation = `
+mutation migrateAppleUser($id: ID!, $email: String!, $transferSub: String!) {
+  migrateAppleUser(id: $id, email: $email, transferSub: $transferSub)
+}`;
+  const variables = { id: id };
```

Comment:
We're missing `email` and `transferSub` variables here and the TS mutation variable is called `deleteMutation` 😅

Reply:
Not sure if you were waiting on the other PR to be merged and then add these, but I've approved that one!

Reply:
😁 thanks for catching it! added the variables and nit change on query variable name

ya this can go - will test after INFRA-1312 is complete